lizard_damage/migrations/0025_auto__add_field_damagescenario_ahn_version.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
```

Comment:
Oermf! Is dit nog south? Daar moeten we een keer wat aan doen. Dat had ik niet door. (Hoort niet in dit pull request, hoor).